### ISSUE TITLE
translate string values in .properties files

### ISSUE THREAD

**Issue by JonDev17** [PSeminarKryptographie/MonkeyCrypt#92]
In order to make all contents available in different languages it is necessary to also translate the strings that are displayed in the application aside from learning or support texts.

If you like to contribute by editing or adding .properties files go to **src / text / local**.
Please follow the convention of naming the files like **Bundle_<language-code>_<country-code>.properties**.
You may copy and paste the content of the Bundle_en_GB.properties and change the string on the **right** side of the key-value declaration.

Up to this point all files except the German and English ones are incomplete. So if you want to change them dive right in!

**Comment by smosgr** [comment]
Do you require Greek or Spanish?

**Comment by JonDev17** [comment]
@smosgr Of course! I prepared the language files for Greek (el_GR) and Spanish (es_ES) for you.

**Comment by smosgr** [comment]
PR raised :)

**Comment by Ph0tonic** [comment]
Hi,
I made you a French translation.

**Comment by tradfursten** [comment]
I'm starting on a Swedish translation

**Comment by mahewi** [comment]
I can add Finnish translation

**Comment by MatheusMuriel** [comment]
I'm starting on a Brazilian Portuguese translation

**Comment by avijit1258** [comment]
#118 Bengali translation complete.

**Comment by GEveloper** [comment]
I will translate to Georgian!
